ui/components/ui/popover/popover.component.js

```diff
@@ -19,15 +46,18 @@ const Popover = ({
   CustomBackground,
   popoverRef,
   centerTitle,
-  headerProps = {},
-  contentProps = {},
-  footerProps = {},
+  headerProps = defaultHeaderProps,
+  contentProps = defaultContentProps,
+  footerProps = defaultFooterProps,
 }) => {
   const t = useI18nContext();
   const showHeader = title || onBack || subtitle || onClose;
   const Header = () => {
     return (
-      <Box {...headerProps} className="popover-header">
+      <Box
+        {...{ ...defaultHeaderProps, ...headerProps }}
```

Comment:
[nit] You shouldn't have to spread the spread; once should be enough:

```suggestion
        { ...defaultHeaderProps, ...headerProps }
```

Reply:
That doesn't work, it gives an "expression expected" error.

Reply:
That doesn't work, it gives an "expression expected" error.